managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerFactoryImpl.java

```diff
@@ -395,6 +396,8 @@ public void initializeComplete() {
 
                     // May need to update the cursor position
                     newledger.maybeUpdateCursorBeforeTrimmingConsumedLedger();
+                    // May need to trigger offloading
+                    newledger.maybeOffloadInBackground(NULL_OFFLOAD_PROMISE);
```

Comment:
Just a suggestion, maybe trigger offloading in BrokerService#createPersistentTopic is better?
Trigger offloading after PersistentTopic load finished:
![image](https://github.com/apache/pulsar/assets/21362791/4c9119f0-8bf3-45f5-9d02-fdf658adbbdc)

Reply:
Offload is a managedledger level feature and does not need to be exposed as the topic-level feature. From the feature scope perspective, we'd better keep it at the managedledger level.

Reply:
It makes sense, and it seems it's OK even load topic failed